static/templates/authentication/register.html

```diff
@@ -74,8 +74,9 @@
         you agree to provide Daemo accurate and complete information about you and your business, and you authorize Daemo to share it and transaction
         information related to your use of the payment processing services provided by Stripe. Daemo does not store
         any banking or credit card information.</div-->
-    <div class="authentication-error" layout="row" ng-show="register.error && form.$pristine"
-         ng-bind="register.error"></div>
+    <div class="authentication-error" layout="row" ng-show="register.error && form.$pristine">
+      <span>We are currently in closed beta. Please <a href="https://goo.gl/forms/8eNjaR8pKc3aIpci1">fill out this form</a> to register interest in using Daemo.</span>
```

Comment:
register error can be anything here, can you check in angular if the response code is 403 or whatever you get when registration closed exception is raised or worst case you can do a string comparison of current error message